test/fluentd-forward.sh

```diff
@@ -61,6 +61,10 @@ update_current_fluentd() {
       </match>' | oc replace -f -
         oc patch configmap/logging-fluentd --type=json --patch '[{ "op": "add", "path": "/data/secure-forward1.conf", "#": "generated config file secure-forward1.conf" }]' 2>&1
         oc patch configmap/logging-fluentd --type=json --patch '[{ "op": "replace", "path": "/data/secure-forward1.conf", "value": "\
+  <filter **>\n\
```

Comment:
Sorry, I missed this earlier - this won't work - you cannot have a `<filter>` inside of a `<match>` block - this oc patch is putting the patch inside a `<match>` block - you might need a separate oc patch in order to edit fluent.conf to add the filter